src/Physics/InitialField.cpp

```diff
@@ -104,6 +108,48 @@ void seissol::physics::SuperimposedPlanarwave::evaluate( double time,
   }
 }
 
+seissol::physics::TravellingWave::TravellingWave(const CellMaterialData& materialData, const TravellingWaveParameters& travellingWaveParameters)
+  //Set phase to 0.5*M_PI, so we have a zero at the origin
+  //The wave travels in direction of kVec
+  //2*pi / magnitude(kVec) is the wave length of the wave
+  : Planarwave(materialData, 0.5*M_PI, travellingWaveParameters.kVec, travellingWaveParameters.varField, travellingWaveParameters.ampField),
+  //origin is a point on the wavefront at time zero
+    m_origin(travellingWaveParameters.origin)
+{
```

Comment:
I don't like this part. Hell is going to break loose if the material is not homog. Maybe add a warning to evaluate()
It probably doesn't make sense to implement this for non-homog. materials.

Reply:
I think "hell is breaking loose" is a bit overstated here, see attached video. There I've set a higher velocity in the middle and a lower velocity at the upper and lower ends. As you see, the initial condition is as it is supposed to be. Also the wave propagation makes sense to me. Only the error computations at the end do not make sense.
![output](https://user-images.githubusercontent.com/24592027/128473971-84694613-40f0-427f-8316-08dd9791974c.gif)

Reply:
Hm, shouldn't you use consider the material of each given cell? Or doesn't this lead to an inconsistent init?